src/ui/resources/settings/whatsnew.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="15" viewBox="0 0 14 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Keep this part out. @flozia and @lesleyjanenorton are working on it.

Reply:
This is just the icon for the DevMenu->Dev_FeatureFlag. Any other icon prefrences ? :D